pkg/util/docker/event_pull.go

```diff
@@ -43,6 +43,7 @@ func (d *DockerUtil) openEventChannel(since, until time.Time, filter map[string]
 func (d *DockerUtil) processContainerEvent(msg events.Message) (*ContainerEvent, error) {
 	// Type filtering
 	if msg.Type != "container" {
+		log.Warnf("Event skipped because of a type mismatch. Expected 'container' got %s", msg.Type)
```

Comment:
This does not warrant a warning, or even a log, we can simply silently ignore then